### PR TITLE
Update Text-SimpChinese-2052.txt

### PR DESCRIPTION
**Pull request by nkh0472**
1. Improve translation for strings: 3220, 3864.
2. New translation for strings: 3305, 3467, 3484, 3821, 3831, 3841, 3851, 3861, 3871, 3961, 3881, 3891, 3911, 3921, 3931, 3951, 3941, 4104, 4105, 4208, 4300, 4301, 4341, 6001, 6002, 6003, 6004, 6005, 6006, 6007.
3. Synchronizes strings 3542, 3543, 3946, 3947, 7866 to the commented status from the English msg file.

### CODE REVIEW

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 1]
```diff
@@ -2550,7 +2595,6 @@ KB
 
 #----------------------------------------------------------------------------
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2597]: Empty line to remove.

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 2]
```diff
@@ -2784,6 +2828,12 @@ COM 访问 (OpenClsid)
 沙盘化的程序无法和沙盘外运行程序的 COM 对象通讯，除非 COM 类 ID 符合以下的类 ID 之一。
 .
 
+4104;txt;01
+资源访问::网络访问
+.
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2833]: Add a new line after the dot.

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 3]
```diff
@@ -2784,6 +2828,12 @@ COM 访问 (OpenClsid)
...
+4105;txt;01
+网络访问 (防火墙)
+.
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2836]: Add a new line after the dot.

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 4]
```diff
@@ -2949,6 +3002,9 @@ PDF 和打印工具
 打开简单服务发现协议（SSDP、UPnP）RPC 端口
 .
 
+4300;txt;01
+打开附加 RPC 端口绑定
+.
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:3007]: Add a new line after the dot.

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 5]
```diff
@@ -3179,6 +3238,9 @@ Sandboxie 配置有密码保护。请输入密码。
 允许直接访问 %2 的会话管理
 .
 
+4341;txt;01
+允许直接访问 %2 笔记
+.
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:3243]: Add a new line after the dot.

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 6]
```diff
@@ -3311,6 +3373,13 @@ BT 种子（Torrent）客户端
 移至(&T)
 .
 
+#----------------------------------------------------------------------------
+# Plus Tips
+#----------------------------------------------------------------------------
+
+6004;pop;err;01
+不允许使用所选的框配置。
+.
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:3382]: Add a new line after the dot.

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 7]
```diff
@@ -960,6 +960,9 @@ Sandboxie Control: %2 版本
 即将中止。
 .
 
+# 3305;txt;01
+# 无法安装托盘图标。即将中止。
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:965]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 8]
```diff
@@ -1279,6 +1282,9 @@ Sandboxie Control: %2 版本
 资源访问(&R)
 .
 
+3484;txt;01
+资源访问
+.
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:1287]: Add a new line after the dot.

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 9]
```diff
@@ -2165,6 +2171,9 @@ Sandboxie 可以在沙盘化程序的窗口边缘显示一条粗线边界。
 # Sandboxie Control:  Sandbox Settings:  Recovery
 #----------------------------------------------------------------------------
 
+# 3821;txt;01
+# 恢复::快速恢复
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2176]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 10]
```diff
@@ -2205,6 +2214,9 @@ Sandboxie 可以在沙盘化程序的窗口边缘显示一条粗线边界。
 # Sandboxie Control:  Sandbox Settings:  Delete
 #----------------------------------------------------------------------------
 
+# 3841;txt;01
+# 删除::删除调用
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2219]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 11]
```diff
@@ -2233,6 +2245,9 @@ Sandboxie 可以在沙盘化程序的窗口边缘显示一条粗线边界。
 从不移除此沙盘或删除其保存的内容
 .
 
+# 3851;txt;01
+# 删除::删除命令
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2250]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 12]
```diff
@@ -2282,6 +2297,9 @@ Sandboxie 可以在沙盘化程序的窗口边缘显示一条粗线边界。
 # Sandboxie Control:  Sandbox Settings:  Forced Folders/Programs
 #----------------------------------------------------------------------------
 
+# 3861;txt;01
+# 程序启动::强制运行文件夹
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2302]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 13]
```diff
@@ -2516,6 +2558,9 @@ KB
 # Sandboxie Control:  Sandbox Settings:  Drop Rights
 #----------------------------------------------------------------------------
 
+# 3941;txt;01
+# 限制::降低权限
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2563]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 14]
```diff
@@ -2500,6 +2539,9 @@ KB
 # 允许此沙盘中的程序管理硬件设备配置
 # .
 
+# 3951;txt;01
+# 限制::硬件访问
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2544]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 15]
```diff
@@ -2472,6 +2508,9 @@ KB
 # Sandboxie Control:  Sandbox Settings:  Low Level Access / Hardware Access
 #----------------------------------------------------------------------------
 
+# 3931;txt;01
+# 限制::底层访问
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2513]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 16]
```diff
@@ -2444,6 +2477,9 @@ KB
 要管理网络访问限制，请打开沙盘设置窗口，并展开限制分组。
 .
 
+# 3921;txt;01
+# 限制::启动/运行 访问
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2482]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 17]
```diff
@@ -2306,6 +2324,9 @@ Sandboxie 可以在沙盘化程序的窗口边缘显示一条粗线边界。
 选择一个文件夹添加到强制运行的文件夹列表中。
 .
 
+# 3871;txt;01
+# 程序启动::强制运行程序
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2329]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 18]
```diff
@@ -2314,6 +2335,9 @@ Sandboxie 可以在沙盘化程序的窗口边缘显示一条粗线边界。
 如果以下任何程序在沙盘外启动, 将强制其在此沙盘中运行。 但如果其已在其他沙盘中启动, 则不适用此项。
 .
 
+# 3961;txt;01
+# 程序启动::强制用户
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2340]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 19]
```diff
@@ -2334,6 +2358,9 @@ Sandboxie 可以在沙盘化程序的窗口边缘显示一条粗线边界。
 # Sandboxie Control:  Sandbox Settings:  Linger/Leader Programs
 #----------------------------------------------------------------------------
 
+# 3881;txt;01
+# 程序停止::驻留程序
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2363]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 20]
```diff
@@ -2346,6 +2373,9 @@ Sandboxie 可以在沙盘化程序的窗口边缘显示一条粗线边界。
 以下程序将自动终止, 如果其在其他所有程序结束后仍在沙盘中运行。
 .
 
+# 3891;txt;01
+# 驻留程序::先导程序
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2378]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 21]
```diff
@@ -2400,6 +2430,9 @@ KB
 # Sandboxie Control:  Sandbox Settings:  Internet/Start Restrictions
 #----------------------------------------------------------------------------
 
+# 3911;txt;01
+# 限制::网络访问
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2435]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 22]
```diff
@@ -2816,6 +2866,9 @@ COM 访问 (OpenClsid)
 选择以下列表中的应用程序以改进它们在 Sandboxie 中的使用。
 .
 
+# 4208;txt;01
+# 应用程序::文件夹
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:2871]: Add a new line after `# .`

File: Sandboxie/msgs/Text-SimpChinese-2052.txt [hunk 23]
```diff
@@ -3095,6 +3151,9 @@ Sandboxie 配置有密码保护。请输入密码。
 # Applications - Accessibility
 #----------------------------------------------------------------------------
 
+# 4301;txt;01
+# 应用程序::可访问性
+# .
```
Comment (isaak654) [inline Sandboxie/msgs/Text-SimpChinese-2052.txt:3156]: Add a new line after `# .`

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Review by isaak654 — CHANGES_REQUESTED** [review]
I would comment out 7923, 7941, 7945, 7946. 
I don't see any reference for these in the source code, except in the same .txt localization files, for example see https://github.com/sandboxie-plus/Sandboxie/search?q=7923

In case you're looking to comment out more lines, you could take a look to my changes [here](https://github.com/sandboxie-plus/Sandboxie/blob/364c43f29bb4f4a4b667d393a5e396b93c6c62ef/Sandboxie/msgs/Text-Italian-1040.txt)

**Comment by nkh0472** [comment]
> In case you're looking to comment out more lines, you could take a look to my changes [here](https://github.com/sandboxie-plus/Sandboxie/blob/364c43f29bb4f4a4b667d393a5e396b93c6c62ef/Sandboxie/msgs/Text-Italian-1040.txt)

Since I found some differences between the English and Italian translation file, and the project are written in English, I'll decide to keep my translation as same as the [English msg file](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/Sbie-English-1033.txt).

If you want to comment out more deprecated strings, it may be better to merge commented lines in both languages, then comment out (or delete?) all those lines in all language files at once.

<details>
<summary>diff between En and It</summary>

```
diff -r X:\en_sta.txt X:\it_sta.txt
9c9
< 1109;evt;err;01
---
> # 1109;evt;err;01
24c24
< 1202;pop;err;01
---
> # 1202;pop;err;01
35c35
< 1222;pop;err;02
---
> 1222;pop;err;01
43c43
< 1303;pop;inf;01
---
> # 1303;pop;inf;01
49c49
< 1310;pop;inf;01
---
> # 1310;pop;inf;01
51,52c51,52
< 1312;pop;inf;01
< 1313;pop;inf;01
---
> # 1312;pop;inf;01
> # 1313;pop;inf;01
198c198
< 3231;txt;01
---
> # 3231;txt;01
200,207c200,207
< 3233;txt;01
< 3234;txt;01
< 3235;txt;01
< 3236;txt;01
< 3237;txt;01
< 3238;txt;01
< 3239;txt;01
< 3240;txt;01
---
> # 3233;txt;01
> # 3234;txt;01
> # 3235;txt;01
> # 3236;txt;01
> # 3237;txt;01
> # 3238;txt;01
> # 3239;txt;01
> # 3240;txt;01
276c276
< 3455;txt;01
---
> # 3455;txt;01
279c279
< 3458;txt;01
---
> # 3458;txt;01
466c466
< 3771;txt;01
---
> # 3771;txt;01
491d490
< # 3821;txt;01
495d493
< # 3831;txt;01
502d499
< # 3841;txt;01
510d506
< # 3851;txt;01
521d517
< # 3861;txt;01
528d523
< # 3871;txt;01
531d525
< # 3961;txt;01
536d529
< # 3881;txt;01
540d532
< # 3891;txt;01
551d542
< # 3911;txt;01
562d552
< # 3921;txt;01
568,580c558,567
< # 3931;txt;01
< # 3932;txt;01
< # 3933;txt;01
< # 3934;txt;01
< # 3935;txt;01
< # 3936;txt;01
< # 3937;txt;01
< # 3938;txt;01
< # 3951;txt;01
< # 3952;txt;01
< # 3953;txt;01
< # 3954;txt;01
< # 3941;txt;01
---
> 3932;txt;01
> 3933;txt;01
> 3934;txt;01
> 3935;txt;01
> 3936;txt;01
> 3937;txt;01
> 3938;txt;01
> 3952;txt;01
> 3953;txt;01
> 3954;txt;01
585,586c572,573
< # 3946;txt;01
< # 3947;txt;01
---
> 3946;txt;01
> 3947;txt;01
588,591d574
< 3950;txt;01
< 3951;txt;01
< 3952;txt;01
< 3953;txt;01
652d634
< # 4208;txt;01
713d694
< # 4301;txt;01
788,828c769,809
< 7901;txt;01
< 7902;txt;01
< 7903;txt;01
< 7921;txt;01
< 7922;txt;01
< 7923;txt;01
< 7924;txt;01
< 7925;txt;01
< 7926;txt;01
< 7927;txt;01
< 7931;txt;01
< 7932;txt;01
< 7933;txt;01
< 7934;txt;01
< 7935;txt;01
< 7936;txt;01
< 7937;txt;01
< 7941;txt;01
< 7942;txt;01
< 7943;txt;01
< 7944;txt;01
< 7945;txt;01
< 7946;txt;01
< 7951;txt;01
< 7952;txt;01
< 7961;txt;01
< 7962;txt;01
< 7971;txt;01
< 7972;txt;01
< 7973;txt;01
< 7974;txt;01
< 7975;txt;01
< 7976;txt;01
< 7977;txt;01
< 7978;txt;01
< 7982;txt;01
< 7983;txt;01
< 7984;txt;01
< 7985;txt;01
< 7986;txt;01
< 7987;txt;01
---
> # 7901;txt;01
> # 7902;txt;01
> # 7903;txt;01
> # 7921;txt;01
> # 7922;txt;01
> # 7923;txt;01
> # 7924;txt;01
> # 7925;txt;01
> # 7926;txt;01
> # 7927;txt;01
> # 7931;txt;01
> # 7932;txt;01
> # 7933;txt;01
> # 7934;txt;01
> # 7935;txt;01
> # 7936;txt;01
> # 7937;txt;01
> # 7941;txt;01
> # 7942;txt;01
> # 7943;txt;01
> # 7944;txt;01
> # 7945;txt;01
> # 7946;txt;01
> # 7951;txt;01
> # 7952;txt;01
> # 7961;txt;01
> # 7962;txt;01
> # 7971;txt;01
> # 7972;txt;01
> # 7973;txt;01
> # 7974;txt;01
> # 7975;txt;01
> # 7976;txt;01
> # 7977;txt;01
> # 7978;txt;01
> # 7982;txt;01
> # 7983;txt;01
> # 7984;txt;01
> # 7985;txt;01
> # 7986;txt;01
> # 7987;txt;01
```
</details>

**Review by isaak654 — APPROVED** [review]

**Comment by isaak654** [comment]
> If you want to comment out more deprecated strings, it may be better to merge commented lines in both languages

So far, this has already been completed for English, Italian, Russian and German languages.

> then comment out (or delete?) all those lines in all language files at once.

It should preferably be done by the translator itself, so for Simplified Chinese you may want to look at [Report-SimpChinese.txt](https://github.com/sandboxie-plus/Sandboxie/blob/master/Sandboxie/msgs/report/Report-SimpChinese.txt)

**Pull request merged**